src/editor/codeeditor.cpp

```diff
@@ -78,18 +84,72 @@ CodeEditor::CodeEditor(QWidget *parent) : QPlainTextEdit(parent)
     fmtComment.setForeground(Qt::green);
     fmtUndefined.setUnderlineStyle(QTextCharFormat::WaveUnderline);
     fmtUndefined.setUnderlineColor(Qt::red);
-    fmtRegular.setForeground(Qt::black);
     fmtLiteral.setForeground(mConfigParam.mStringsColor);
     fmtKeyword.setForeground(mConfigParam.mBasicLiteralsColor);
     fmtComment.setForeground(mConfigParam.mCommentColor);
     fmtRegular.setForeground(mConfigParam.mCodeTextColor);
+    fmtRegular.setForeground(Qt::white);
+}
+
+void CodeEditor::handleLinesAddition(int changeStart, int lastLineWithChange, int lineDifference)
```

Comment:
You can pass here a const or const& value to avoid some unnecessary changes
the same in function below